pyparsing/core.py

```diff
@@ -2155,7 +2163,6 @@ def run_tests(
                 t = NL.transform_string(t.lstrip(BOM))
                 result = self.parse_string(t, parse_all=parseAll)
             except ParseBaseException as pe:
-                fatal = "(FATAL)" if isinstance(pe, ParseFatalException) else ""
```

Comment:
Rejected - instead of deleting this, make line below an f-string like: `out.append(f"FAIL{fatal}: {pe}")`